src/js/tabs/osd.js

```diff
@@ -3325,6 +3325,7 @@ osd.cleanup = function(callback) {
 };
 
 window.TABS.osd = osd;
+window.OSD = OSD;
```

Comment:
Maybe is better to modify in the VirtualFC file:
https://github.com/betaflight/betaflight-configurator/blob/db99a988073a5f0f6c87ec00aa5d31b6c618b7a4/src/js/VirtualFC.js#L179

Modify `const virtualOSD = OSD` by `const virtualOSD = TABS.osd`? In this way we don't duplicate here the variable. I did a little test yesterday but it not worked, but maybe I did something wrong.

Reply:
Good shout. Will grab a look

Reply:
Can we use export / import?

Reply:
Eventually we can, but would need move virtual fc to modules first. But at the moment I'm trying to do similar thing to GUI first.

Reply:
@chmelevskij any update? Do you like to have this merged as temporary fix?